docs/general/administration/hardware-selection.md

```diff
@@ -103,7 +103,11 @@ For decoding support, the more codecs the better. However, there are only a few
 
 Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
 
-Do Note that Intel ARC requres kernel version 6.2 or newer on Linux. Please check your distribution for details.
+:::note Intel 12th Gen, 13th Gen and ARC Support
+
+If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that jellyfin-ffmpeg6 be installed instead of the default jellyfin-ffmpeg5 on these hardware platforms.
```

Comment:
```suggestion
If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that `jellyfin-ffmpeg6` be installed instead of the default `jellyfin-ffmpeg5` on these hardware platforms.
```